byway/src/main/java/com/google/sps/data/Rec.java

```diff
@@ -12,19 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- *
- */
-/* exported addRandomGreeting */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+package com.google.sps.data;
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/** A sample class to mock data received from another datastore table*/
+// TODO: Make a parent class that Rec and Stop can extend 
+public final class Rec{
```

Comment:
Rec? what is rec? Please avoid abbreviations in things like class names.

Reply:
Okay, will change to "Recommendation" for clarity.